packages/client/src/SubmissionController.ts

```diff
@@ -78,6 +82,9 @@ const ACTION_LIST: IActionList = {
   [Action.APPROVE_APPLICATION]: Action.APPROVE_APPLICATION,
   [Action.REGISTER_APPLICATION]: Action.REGISTER_APPLICATION,
   [Action.REJECT_APPLICATION]: Action.REJECT_APPLICATION,
+  [Action.ARCHIVE_APPLICATION]: Action.ARCHIVE_APPLICATION,
+  [Action.REINSTATE_APPLICATION]: Action.REINSTATE_APPLICATION,
+  [Action.COLLECT_CERTIFICATE]: Action.COLLECT_CERTIFICATE,
   [Action.ARCHIVE_DECLARATION]: Action.ARCHIVE_DECLARATION,
   [Action.COLLECT_CERTIFICATE]: Action.COLLECT_CERTIFICATE,
```

Comment:
`COLLECT_CERTIFICATE` is mapped twice here

Reply:
Removed